docs/current_docs/api/packages.mdx

```diff
@@ -17,6 +17,20 @@ To add a Go module, add it to your `go.mod` file using `go get`. For example:
 go get github.com/spf13/cobra
 ```
 
+Dagger also lets you import [private Go modules](https://go.dev/ref/mod#private-modules) in your Dagger module. To do this, add the URL to your private repository(ies) in your `dagger.json` file, as shown below:
```

Comment:
@rajatjindal can you specify multiple repositories? If yes, how?

Reply:
Assuming this works with non-GitHub repositories also?

Reply:
yes, for any url (or a prefix of that) that can be specified as dependency in golang code.